plugin/audit_log/audit_log.cc

```diff
@@ -1711,7 +1713,11 @@ static st_mysql_audit audit_log_descriptor = {
 */
 
 static SHOW_VAR audit_log_status_variables[] = {
-    {NullS, NullS, SHOW_LONG, SHOW_SCOPE_GLOBAL}};
+    {"Audit_log_buffer_size_overflow",
+     (char*) &audit_log_buffer_size_overflow,
```

Comment:
This is a bit complicated to get absolutely right as SHOW_VAR expects a regular not atomic var. Strictly speaking we should have a function, which would do atomic load, but maybe we can get aways with static_assert(sizeof(type) == sizeof(std::atomic<type>)); next to this instead

Reply:
@laurynas-biveinis `SHOW_LONG` is 4 bytes and `std::atomic<uint64_t>` is 8 bytes.
Shall we use  std::atomic<uint32> ?

Reply:
Never mind, I did the function with atomic load path.